.github/ISSUE_TEMPLATE/request-package-removal.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: Request a package to be removed
+description: Request removal of a package from the Solus repository. The package maybe unmaintained, replaced, deprecated or other reason.
+labels:
+  - Package: Removal Request
```

Comment:
Does this have to be in quotes?

Reply:
From their docs:
`labels` (Array or String): This issue will automatically receive these labels upon creation. Can be array of labels or comma-delimited string, e.g. "bug,needs-triage"

Probably needs quotes?

Reply:
Just tested it [here](https://github.com/getsolus/packages/commit/422ae16986f7878b2ca9112314bd08c37f0cf0d7). It needs quotes.